services/ui-src/src/utils/other/mask.tsx

```diff
@@ -0,0 +1,60 @@
+/*
+ * Custom Masks Type Guard
+ * Add any future custom masks here!
+ */
+const customMasks = ["comma-separated"] as const;
+export type CustomMasks = typeof customMasks[number];
+export const isCustomMask = (x: any): x is CustomMasks =>
+  customMasks.includes(x);
```

Comment:
I wonder if instead of resorting to types, this could just use the same kind of pattern as we use for the form factory. Something like: 

``` 
const customMaskMap: any = {
    "commaSeparated": CommaSeparatedMaskGoesHere,
    "futureMask1": FutureMask1GoesHere,
    ...and so on
  };

// then access like this
const maskToUse = customMaskMap[<string key from field props>] || fallbackThing;
```

Would also rename to something like `maskToUse` or `selectedMask` or something, 'cause what we're getting back from it is not just "is it a custom mask" but the actual mask that we want to use.